### PR TITLE
MultiFlite nano brushed FC target

### PR DESCRIPTION
**Pull request by EvilBadger**
Adds support for new board type [MFBC]

Broken out of original pull request [1/3]

### CODE REVIEW

File: src/main/target/MFNBFC/target.h [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "MFNBFC" // MultiFlite Nano Brushed FC
```
Comment (hydra) [inline src/main/target/MFNBFC/target.h]: 4 characters max

**Comment by hydra** [comment]
would perhaps be better if this is split up into 3 PRs
1) target support
2) vtx support
3) battery divider resistor configuration

**Comment by EvilBadger** [comment]
Split commit in to two parts (target and battery), will add vtx support in the near future as a separate pull request.

**Comment by EvilBadger** [comment]
Hi, any update on when this may get pulled in or what more needs to be done to it?

**Comment by EvilBadger** [comment]
HI, 

I have resolved all of the conflicts, is it possible to get this branch merged?

Thanks

**Comment by hydra** [comment]
Since the board appears to be the same as a naze but with different components it would make sense to use the naze target for the board if possible.  I'm guessing the board does not have and will never have all of the sensors that you added in the source file list on the Makefile.  The AlienWiiF1 is a variant of the naze.  It seems that this board is no different in that regard and should probably work the same way unless there is a good technical reason not to.

there's no need to have target specific support for NPN buzzer, see BEEPER_INVERTED.

**Comment by hydra** [comment]
Closing, no response or update.

**Pull request closed without merging**